Reconstruction/RecTracker/src/components/CombinatorialSeedingTest.h

```diff
@@ -0,0 +1,40 @@
+#ifndef RECTRACKER_COMBINATORIALSEEDINGTEST_H
+#define RECTRACKER_COMBINATORIALSEEDINGTEST_H
+
+// GAUDI
+#include "GaudiAlg/GaudiAlgorithm.h"
+#include "GaudiKernel/RndmGenerators.h"
+#include "GaudiKernel/ToolHandle.h"
+
+// FCCSW
+#include "FWCore/DataHandle.h"
+
+class IGeoSvc;
+class ITrackSeedingTool;
+
+namespace fcc {
+class TrackHitCollection;
+class PositionedTrackHitCollection;
+}
+
+class CombinatorialSeedingTest : public GaudiAlgorithm {
+public:
+  CombinatorialSeedingTest(const std::string& name, ISvcLocator* svcLoc);
+
+  ~CombinatorialSeedingTest();
+
+  StatusCode initialize();
+
+  StatusCode execute();
+
+  StatusCode finalize();
```

Comment:
Can you add `override final` here and remove the trivial dtor implementation adding just `= default` here?